### ISSUE TITLE
Add option to use adb from system path instead of Sdk/

### ISSUE THREAD

**Issue by Emaleth** [godotengine/godot-proposals#9495]
### Describe the project you are working on

any

### Describe the problem or limitation you are having in your project

NixOS, by default, can't run normal linux binaries. While exporting to android is not an issue, one click deploy, which relies on adb inside Sdk path is. 

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Since there is an adb package, using adb from system path (like in terminal when you just type adb [comand]), would solve the issue.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

just a toggle inside android export editor settings "Use adb from system path".

### If this enhancement will not be used often, can it be worked around with a few lines of script?

i tried wrapping adb executable in a script but to no effect.

### Is there a reason why this should be core and not an add-on in the asset library?

no idea.

**Comment by Calinou** [comment]
I don't think this needs to be an option. We can fall back to system `adb` in `PATH` automatically if adb isn't found (or can't be run) in the SDK folder.

**Comment by Emaleth** [comment]
> I don't think this needs to be an option. We can fall back to system `adb` in `PATH` automatically if adb isn't found (or can't be run) in the SDK folder.

Even better.
